tools/java/org/unicode/cldr/util/LanguageTagCanonicalizer.java

```diff
@@ -52,6 +52,9 @@ public LanguageTagCanonicalizer(LstrType lstrType) {
     // TODO, handle variants
     @Override
     public synchronized String transform(String locale) {
+        return transform (locale, OutputOption.ICU_LCVARIANT);
+    }
+    public synchronized String transform(String locale, OutputOption oo) {
```

Comment:
"oo" doesn't seem like a particularly clear variable name to me. Maybe "outputOpts" ?

Reply:
I'd like to clean this up later. To much to do right now, and I don't want another cycle.